microsoft/ReportingServicesTools
# Create Get-RsRestItem.ps1

Initial commit of new Get-RsRestItem function.

Fixes # 
None.

Changes proposed in this pull request:
Added new Get-RsRestItem function.

How to test this code:
 - See Help examples.
 - `Get-RsRestItem -RsItem "/MyReport"`
 - `Get-RsRestItem -RsItem "/Finance"`

Has been tested on (remove any that don't apply):
 - PowerShell 5.1 and above
 - Windows 10
 - Power BI Report Server
